docs/docs/etl-pipeline-tutorial/0-extract-data.md

```diff
@@ -8,48 +8,63 @@ In the first step in our pipeline, we will use [software-defined assets](/guides
 
 ## 1. Scaffold an asset
 
-
 We will start by writing our ingestion assets. Assets serve as the building blocks for our platform in Dagster and represent the underlying entities in our pipelines (such as tables, datasets, or machine learning models). Assets take dependencies on other assets to build out the full graph of our data platform and define its lineage.
 
-
 When building assets, the first step is to scaffold the assets file with the [`dg scaffold` command](/api/dg/dg-cli#dg-scaffold). The `dg` CLI provides a number of commands to help structure and navigate Dagster projects. For more information, see the [`dg` CLI documentation](/api/dg/dg-cli):
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/commands/dg-scaffold-assets.txt" />
 
 This adds a file called `assets.py` that will contain our asset code to the `etl_tutorial` module. Using `dg` to create the file ensures that the file is in a location where it can be automatically discovered by Dagster:
 
 <CliInvocationExample path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/tree/assets.txt" />
-   
 ## 2. Write DuckDB helper functions
 
 Since we will be working with DuckDB, we will need to add the DuckDB Python library to our Dagster project:
 
-<CliInvocationExample contents="uv pip install duckdb" />
+<Tabs groupId="package-manager">
+   <TabItem value="uv" label="uv">
+      Install the required dependencies:
+
+         ```shell
+         uv pip install duckdb
```

Comment:
Sorry to keep having feedback on installs, but do you think we can do `uv add` here, or is that not compatible?

Reply:
I'm fine with using `add`. Was mostly just matching the syntax of the quick start

@neverett Do you want me to create and assign myself a ticket to update our instances of `uv pip install` to `uv add`?

Reply:
@dehume yeah, I think a followup ticket makes sense here, thanks!